src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -676,6 +676,8 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             'Populate the target framework combobox
             PopulateTargetFrameworkComboBox(Me.TargetFramework)
+
+            Me.AssemblyInfoButton.Visible = Not Utils.IsTargetingDotNetCore(Me.ProjectHierarchy)
```

Comment:
Made the change and opened #1001 to track the other bug. Once that's implemented the button will be disabled.